src/Facebook/InstantArticles/Transformer/Rules/Rule.php

```diff
@@ -8,72 +8,87 @@
  */
 namespace Facebook\InstantArticles\Transformer\Rules;
 
+use Facebook\InstantArticles\Elements\Element;
+use Facebook\InstantArticles\Transformer\Transformer;
+
 abstract class Rule
 {
-    public function matches($context, $node)
+    public function matches(Element $context, \DOMNode $node): bool
     {
-        $log = \Logger::getLogger('facebook-instantarticles-transformer');
-
         $matches_context = $this->matchesContext($context);
         $matches_node = $this->matchesNode($node);
-        if ($matches_context && $matches_node) {
-            $log->debug('context class: '.get_class($context));
-            $log->debug('context matches: '.($matches_context ? 'MATCHES' : 'no match'));
-            $log->debug('node name: <'.$node->nodeName.' />');
-            $log->debug('node matches: '.($matches_node ? 'MATCHES' : 'no match'));
-            $log->debug('rule: '.get_class($this));
-            $log->debug('-------');
-        }
-        if ($node->nodeName === 'iframe') {
-            $log->debug('context class: '.get_class($context));
-            $log->debug('context matches: '.($matches_context ? 'MATCHES' : 'no match'));
-            $log->debug('node name: <'.$node->nodeName.' />');
-            $log->debug('node: '.$node->ownerDocument->saveXML($node).' />');
-            $log->debug('node matches: '.($matches_node ? 'MATCHES' : 'no match'));
-            $log->debug('rule: '.get_class($this));
-            $log->debug('-------');
-        }
+        // if ($matches_context && $matches_node) {
```

Comment:
Let's get rid of all these commented lines =)

Reply:
Let's also create an issue for adding proper verbosity levels.

Reply:
Indeed: #285